auth/tests/api/users.rs

```diff
@@ -0,0 +1,96 @@
+use crate::helpers::{spawn_app, ADMIN_KEY};
+use serde_json::{self, Value};
+
+#[tokio::test]
+async fn post_user() {
+    let app = spawn_app().await;
+
+    // POST user without bearer token.
+    let response = app
+        .api_client
+        .post(&format!("{}/user/{}", app.address, "test-user"))
+        .send()
+        .await
+        .expect("Failed to execute request.");
+
+    assert_eq!(response.status().as_u16(), 400);
```

Comment:
You should be able to compare against `StatusCode::FORBIDDEN`(?) which will be easier that the code

Reply:
Indeed!